src/openMVG/graph/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
 
-UNIT_TEST(openMVG_graph connectedComponent "lemon")
-UNIT_TEST(openMVG_graph triplet_finder "lemon")
+UNIT_TEST(openMVG_graph connectedComponent "")
```

Comment:
@pmoulon ,

I notice that, BOOST replaced it but traces are still present, please check:
find src/ -type f -exec grep -H 'lemon/' {} \; | grep -v third_party

If agree, to be 'correct' i leave LEMON for now just like it was, i believe we should clean up LEMON in another PR.

--------------<<<------------
Lets see some examples:

A) BOOST replaces just fine:
src/openMVG/multiview/rotation_averaging_l1.cpp

B) boost does not replaces at all:
src/openMVG/graph/graph.hpp
-------------->>>--------------